BedrockServer.cpp

```diff
@@ -1982,6 +2017,11 @@ void BedrockServer::_acceptSockets() {
     // Try block because we sometimes catch `std::system_error` from in here (likely from the thread code) and we're
     // trying to diagnose exactly what's happening.
     try {
+        if (_outstandingSocketThreads >= _maxSocketThreads) {
+            SINFO("Not accepting any new socket threads as we already have " << _outstandingSocketThreads << " of " << _maxSocketThreads);
```

Comment:
Can we have this log line for db handles? Or we already have it?

Reply:
We already have a similar log line.